aerleon/lib/nokiasrl.py

```diff
@@ -0,0 +1,303 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
You can remove the copyright notices if this was created by yourself.
You can see in any code that was not forked from Capirca we have left the header off.

Reply:
I mostly copied from the openconfig file, so I figured I'd leave it in place

Reply:
I actually just noticed this. There is a lot of overlap here. What happens if we find a bug in the openconfig code? We will have to fix it in two places. That isn't a great position to be in. Would you be able to inherit from Openconfig or create a common class for them to inherit from?

Reply:
Sure, I suppose we could create a common base class and then have generator methods for source-address, dest-address, etc. ( i.e. refactor Openconfig and then derive SRLinux from that, overriding those methods )